apps/examples/wifi_test/wifi_test.c

```diff
@@ -248,7 +249,9 @@ int main(int argc, FAR char *argv[])
 int wifi_test_main(int argc, char *argv[])
 #endif
 {
-
+#if CONFIG_TASH
+	tash_cmd_install("wifi_scan", wifi_test_proc, TASH_EXECMD_SYNC);
```

Comment:
Now that the CONFIG_BUILTIN_APPS has been added, you do not need to call tash_cmd_install() here.